content/en/docs/instrumentation/php/automatic.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Automatic Instrumentation
+linkTitle: Automatic
+weight: 2
+---
+
+Automatic instrumentation with PHP requires at least PHP 8.0, and [the opentelemetry PHP extension](https://github.com/open-telemetry/opentelemetry-php-instrumentation).
+The extension allows developers code to hook into classes and methods, and execute userland code before and after.
+
+## Example
+```php
+<?php
+OpenTelemetry\Instrumentation\hook(
+    'class': DemoClass::class,
+    'function': 'run',
+    'pre': static function (DemoClass $demo, array $params, string $class, string $function, ?string $filename, ?int $lineno) use ($tracer) {
+        static $instrumentation;
+        $instrumentation ??= new CachedInstrumentation('example');
+        $instrumentation->tracer()->spanBuilder($class)
+            ->startSpan()
+            ->activate();
+    },
+    'post': static function (DemoClass $demo, array $params, $returnValue, ?Throwable $exception) use ($tracer) {
+        $scope = Context::storage()->scope();
+        $scope->detach();
+        $span = Span::fromContext($scope->context());
+        if ($exception) {
+            $span->recordException($exception);
+            $span->setStatus(StatusCode::STATUS_ERROR);
+        }
+        $span->end();
+    }
+);
+
+$demo = new DemoClass();
+$demo->run();
+```
+
+Here, we provide 'pre' and 'post' functions, which are executed before and after `DemoClass::run`. The `pre` function starts and activates a
+span, and the `post` function ends it.
+
+## Setup
+
+1. Install the extension via pickle or [php-extension-installer](https://github.com/mlocati/docker-php-extension-installer):
+   ```console
+   $ install-php-extensions open-telemetry/opentelemetry-php-instrumentation@main
+   ```
+2. Verify that the extension is installed and enabled:
+   ```console
+   $ php -m | grep  otel_instrumentation
+   ```
+
+## Zero-code auto-instrumentation
```

Comment:
This title is misleading, because if I understand it correctly it is not the instrumentation part that requires zero-code but the configuration
```suggestion
## Zero-code configuration for Auto Instrumentation
```